tekton/publish.yaml

```diff
@@ -130,6 +130,9 @@ spec:
       #!/usr/bin/env sh
       set -ex
 
+      # Use the generated `.ko.yaml`
+      KO_CONFIG_PATH=/workspace/.ko.yaml
```

Comment:
I was surprised to find this accepted a filepath and not the directory.

I was _even more surprised_ to find I reviewed and approved the PR that made it accept being a filepath: https://github.com/ko-build/ko/pull/731 🙃